### ISSUE TITLE
Error when passing default value as options

### ISSUE THREAD

**Issue by DanielHuisman** [gilbsgilbs/babel-plugin-i18next-extract#21]
If the default value of a translation is passed as the second argument to the `t` function an error occurs. Example:
```
t('login.header', 'Sign in')
```

```
/data/Projects/project-frontend/src/pages/login/Login.tsx: Cannot use 'in' operator to search for 'context' in Sign in
    at parseTCallOptions (/data/Projects/project-frontend/node_modules/babel-plugin-i18next-extract/lib/index.js:539:34)
    at extractTCall (/data/Projects/project-frontend/node_modules/babel-plugin-i18next-extract/lib/index.js:573:16)
    at Object.extractTFunction (/data/Projects/project-frontend/node_modules/babel-plugin-i18next-extract/lib/index.js:595:13)
    at CallExpression.collect (/data/Projects/project-frontend/node_modules/babel-plugin-i18next-extract/lib/index.js:1372:32)
    at handleExtraction (/data/Projects/project-frontend/node_modules/babel-plugin-i18next-extract/lib/index.js:1355:16)
    at PluginPass.CallExpression (/data/Projects/project-frontend/node_modules/babel-plugin-i18next-extract/lib/index.js:1369:9)
    at NodePath._call (/data/Projects/project-frontend/node_modules/@babel/traverse/lib/path/context.js:53:20)
    at NodePath.call (/data/Projects/project-frontend/node_modules/@babel/traverse/lib/path/context.js:40:17)
    at NodePath.visit (/data/Projects/project-frontend/node_modules/@babel/traverse/lib/path/context.js:88:12)
    at TraversalContext.visitQueue (/data/Projects/project-frontend/node_modules/@babel/traverse/lib/context.js:118:16)
```

This is supported behaviour according to [the i18next documentation](https://www.i18next.com/translation-function/essentials#passing-a-default-value) and it would be nice if `babel-plugin-18next-extract` also supports this.

**Comment by gilbsgilbs** [comment]
Thanks. I wasn't aware this was allowed. I'm releasing a fix in 0.1.2 tonight.

**Comment by DanielHuisman** [comment]
Great, thanks for the quick response!
